web/src/app/alerts/CreateAlertDialog/StepContent/Step1.js

```diff
@@ -0,0 +1,187 @@
+import React, { useRef } from 'react'
+import {
+  Grid,
+  TextField,
+  InputAdornment,
+  List,
+  ListItem,
+  ListItemText,
+  ListItemIcon,
+  Paper,
+  Typography,
+  InputLabel,
+  Chip,
+} from '@material-ui/core'
+import { makeStyles } from '@material-ui/core/styles'
+import { FormField } from '../../../forms'
+import ServiceLabelFilterContainer from '../../../services/ServiceLabelFilterContainer'
+import { Search as SearchIcon } from '@material-ui/icons'
+import FavoriteIcon from '@material-ui/icons/Star'
+import { ServiceChip } from '../../../util/Chips'
+import AddIcon from '@material-ui/icons/Add'
+
+const useStyles = makeStyles(theme => ({
+  addAll: {
+    backgroundColor: theme.palette.primary['400'],
+  },
+  chipContainer: {
+    padding: theme.spacing(0.5),
+    marginBottom: theme.spacing(2),
+    height: '9em',
+    overflow: 'auto',
+    border: '1px solid #bdbdbd',
+  },
+  endAdornment: {
+    display: 'flex',
+    alignItems: 'center',
+  },
+  noticeText: {
+    width: '100%',
+    textAlign: 'center',
+    alignSelf: 'center',
+    lineHeight: '9em',
+  },
+}))
+
+const CREATE_ALERT_LIMIT = 35
```

Comment:
put this in `app/config/index.js` or at least higher up. it shouldn't be defined in a specific step (esp. since this step isn't the one that makes the API call)